packages/core/admin/admin/src/content-manager/components/EditViewDataManagerProvider/utils/handleAPIError.js

```diff
@@ -0,0 +1,15 @@
+import { getTrad } from '../../../utils';
+
+export function handleAPIError(error) {
```

Comment:
Why not extending the getYupInner util from the helper plugin, it's almost the same code:

```
const getYupInnerErrors = (error, prefix) => {
// ...
  acc[<logic>] = { id: prefix ? prefix + curr.message : curr.message, defaultMessage: curr.message }

  return acc;
 // ...
};
```

Reply:
I'm not in favor of this tbh for two reasons:

- the function is called `getYupInnerErrors` and should therefore only be used for `yup` errors
- it is similar but not equal: both serve different domains and specific formats. As discussed before: we should unify both, but for now the formats are of the API and yup are different.

I renamed the function to `getAPIInnerError` though, to bring it closer to this one and to make sure it doesn't handle the error, but just transforms the data.

What do you think?